x-pack/packages/kbn-elastic-assistant/impl/assistant/assistant_header/index.tsx

```diff
@@ -40,6 +41,8 @@ interface OwnProps {
   title: string | JSX.Element;
   conversations: Record<string, Conversation>;
   refetchConversationsState: () => Promise<void>;
+  anonymizationFields: FindAnonymizationFieldsResponse;
+  refetchAnonymizationFieldsResults: () => Promise<FindAnonymizationFieldsResponse | undefined>;
```

Comment:
nit: Instead of drilling `refetchAnonymizationFieldsResults` it's possible to use invalidation function exposed by a hook like [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/detection_engine/rule_management/api/hooks/use_fetch_rule_by_id_query.ts#L53).